public/javascripts/SVLabel/src/SVLabel/task/Task.js

```diff
@@ -30,6 +30,27 @@ function Task (geojson, tutorialTask, currentLat, currentLng, startPointReversed
         tutorialTask: tutorialTask
     };
 
+    // Observed area radius.
+    const radius = 50;
```

Comment:
You could also maybe group all these things together into like a `fogOfWarProperties` object or something